Assets/scripts/Managers/GameManager.cs

```diff
@@ -116,7 +133,53 @@ void OnDisable()
 		}
 	}
 
-	void RestartRound(){
+    int GetOccuptionsCount(JobType jobType)
+    {
+        int count = 0;
+
+        foreach (var player in PlayerList.playerList.connectedPlayers)
+        {
+            if (player.Value != null)
+            {
+                var mob = player.Value.GetComponent<PlayerScript>();
+                if (mob != null)
+                {
+                    if (mob.JobType == jobType)
+                    {
+                        count++;
+                    }
+                }
+            }
+        }
+
+        return count;
+    }
+
+    public List<string> GetOccupationEquipment(JobType jobType)
+    {
+        return Occuptions.Where(o => o.GetComponent<OccupationRoster>().Type == jobType).First().GetComponent<OccupationRoster>().startingItemHiers;
```

Comment:
Probably doesn't matter for these methods if they only run on start, but in general I would avoid using Linq extension methods at runtime because they generate a lot of garbage object allocations which will impact performance if we let them get out of control.